www/common/inner/common-mediatag.js

```diff
@@ -124,7 +124,17 @@ define([
         return ANIMALS[seed % ANIMALS.length] || '';
     };
 
+    const emojiWithZWJRegex = /(\p{Emoji_Presentation}|\p{Emoji_Modifier_Base})(\p{Emoji_Modifier})?(?:\u200D(\p{Emoji_Presentation}|\p{Emoji_Modifier_Base})(\p{Emoji_Modifier})?)*/gu;
```

Comment:
When using complex regex such as this one, please add a comment about what it does :)
FYI it seems this regex doesn't work in some cases from https://www.unicode.org/Public/emoji/latest/emoji-zwj-sequences.txt (the emoji is split with the regex). e.g. 🧑🏿‍❤️‍💋‍🧑🏽